openaps_reports.py

```diff
@@ -3,6 +3,7 @@
 
 """
 from datetime import datetime
+from dateutil.parser import parse
```

Comment:
Thanks for checking this. I should have reviewed the overall diff vs master before pushing.